package.json

```diff
@@ -100,6 +100,7 @@
     "domhandler": "^4.3.0",
     "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#886f90cbd5e83218fdfd7784d8356c308ef05791",
     "fbjs": "^3.0.2",
+    "focus-trap-react": "^10.2.1",
```

Comment:
Please use v10.2.3 to include https://github.com/focus-trap/focus-trap/issues/1068